### ISSUE TITLE
Changing PDF background color

### ISSUE THREAD

**Issue by Yahllil** [binSaed/flutter_cached_pdfview#83]
Is there a way of changing the PDF's background color (instead of the default grey color)? And is there a way to snap the PDF to the top part of the widget, and avoid the vast "dead area" around the document?

**Comment by SpirikleOfficial** [comment]
Hi, I am also having the same issue of fixed background color. How can we get a way to change it?

**Comment by okbye123** [comment]
This will help https://www.youtube.com/watch?v=gAUVz0U7eyA

**Comment by teamacco** [comment]
I’m interresting to that too

**Comment by bonnybun** [comment]
I just have added this parameter. 
Your can use this package
  ```
flutter_cached_pdfview:
    git:
      url: "https://github.com/bonnybun/flutter_cached_pdfview"
```

A pull request will be created soon, once PDFView project will accept my PR and will be updated

**Comment by FlorianFlatscher** [comment]
Hi, I am interested in this change too. What is the status on your behalf, is there anything stopping you from creating a PR?

**Comment by bonnybun** [comment]
Because flutter_cached_pdfview is based on flutter_pdfview
I have already made a PR  https://github.com/endigo/flutter_pdfview/pull/278
It needs to be merged first before creating a new one for this project

**Comment by bonnybun** [comment]
@FlorianFlatscher https://github.com/endigo/flutter_pdfview/pull/278  has been merged

I have created a new Pull Request
[flutter_cached_pdfview/pull/#110](https://github.com/binSaed/flutter_cached_pdfview/pull/110)

**Comment by FlorianFlatscher** [comment]
@bonnybun Thank you, you are the best 🙏

**Comment by binSaed** [comment]
Solved in #110

`version: 0.4.3`
